_posts/2020-11-08-why-and-how-to-write-good-commit-messages.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title:  "How to and Why Write Good Commit Messages"
```

Comment:
I know exactly what you're trying to say here, but I think it may need some rewording. I would suggest maybe changing it to "How to Write Good Commit Messages" and then explain why its important in your blog post (which I think you end up doing anyway).